.gitignore

```diff
@@ -83,6 +83,10 @@ dist
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+.yarnrc.yml
```

Comment:
where does this file came from in gitignore?

Reply:
these are files generated during th project build, after running 'yarn install'. since they were, untracked files. Never wanted to push them as they might cause some conflicks. Reason I ignored .yarnrc.yml file. Doubt if my approach is the best but had to do it

Reply:
LGTM

Reply:
This file shouldn't change. I've pushed a commit amending that.